contracts/core/EntryPoint.sol

```diff
@@ -667,7 +674,7 @@ contract EntryPoint is IEntryPoint, StakeManager, NonceManager, ReentrancyGuard,
                         }(mode, context, actualGasCost)
                         // solhint-disable-next-line no-empty-blocks
                         {} catch (bytes memory reason) {
```

Comment:
`catch(bytes memory reason)` forces the compiler to allocate and copy the entire returned data, potentially causing OOG.
I wouldn't care if we can verify that it always attributed to the paymaster.
An alternative is use `catch {`, and use `getReturnData(1000)`, to cap the amount of revert data we copy .